packages/elements/src/photon-multirx-form/components/PatientCard.tsx

```diff
@@ -64,89 +65,44 @@ export const PatientCard = (props: {
     }
   });
 
+  const patientId = createMemo(() => props.store['patient']?.value?.id || props?.patientId);
+
   return (
-    <photon-card>
-      <div class="flex flex-col gap-3">
-        <p class="font-sans text-l font-medium">
-          {props?.patientId ? 'Patient' : 'Select Patient'}
-        </p>
-        {/* Show Dropdown when no patientId is passed */}
-        <Show when={!props?.patientId}>
-          <photon-patient-select
-            invalid={props.store['patient']?.error ?? false}
-            help-text={props.store['patient']?.error}
-            on:photon-patient-selected={updatePatient}
-            selected={props.store['patient']?.value?.id ?? props.patientId}
-            sdk={props.client!.getSDK()}
-          />
-        </Show>
-        {/* Show Patient Name when patientId is passed */}
-        <Show when={props?.patientId}>
-          <Show
-            when={store?.selectedPatient?.data?.id}
-            fallback={<sl-spinner style={{ 'font-size': '1rem' }} />}
-          >
-            <p class="font-sans text-gray-700">
-              {store?.selectedPatient?.data?.name?.full},{' '}
-              {formatDate(store?.selectedPatient?.data?.dateOfBirth || '')}
+    <div class="flex flex-col gap-8">
+      <Show when={!props?.patientId}>
+        <photon-card>
+          <div class="flex flex-col gap-3">
+            <p class="font-sans text-l font-medium">
+              {props?.patientId ? 'Patient' : 'Select Patient'}
             </p>
-          </Show>
-        </Show>
-
-        <Show when={props.store['patient']?.value?.id && props.enableOrder && !props.hideAddress}>
+            {/* Show Dropdown when no patientId is passed */}
+            <photon-patient-select
+              invalid={props.store['patient']?.error ?? false}
+              help-text={props.store['patient']?.error}
+              on:photon-patient-selected={updatePatient}
+              selected={props.store['patient']?.value?.id ?? props.patientId}
+              sdk={props.client!.getSDK()}
+            />
+          </div>
+        </photon-card>
+      </Show>
+      <Show when={patientId() && !props.hideAddress}>
+        <photon-card>
+          <PatientInfo
+            patientId={patientId()}
+            weight={props?.weight}
+            editPatient={props?.enableOrder ? () => setDialogOpen(true) : undefined}
+          />
           <photon-patient-dialog
+            hide-create-prescription={true}
```

Comment:
just checking this should always be true

Reply:
Just in this situation where we're setting the flag we'll be falling back to not showing the "create prescription" button (because we're already creating an Rx). 

In other scenarios, like creating a patient, we'll want the option to create a Rx immediately after saving.